eng/Build.props

```diff
@@ -13,8 +13,7 @@
         <ItemGroup>
           <ProjectToBuild Include="$(RepoRoot)Microsoft.DotNet.Cli.sln" />
           <ProjectToBuild Condition="'$(OS)' == 'Windows_NT' And ('$(Architecture)' == 'x86' Or '$(Architecture)' == 'x64' Or '$(Architecture)' == 'arm64' Or '$(Architecture)' == 'arm')"
-                          Include="$(RepoRoot)eng\version.csproj;
-                                   $(RepoRoot)eng\native.proj" />
+                          Include="$(RepoRoot)src\finalizer\finalizer-build.csproj" />
```

Comment:
This was the primary simplification to the finalizer build project chain. The csproj handles all the build logic prior to the .nativeproj with with CMake SDK.

Reply:
yeah Windows arm32 was removed in .NET5: https://github.com/dotnet/installer/issues/8624

Reply:
Okay. I've removed the `arm` condition both here and where I have this logic in the SDK repo's PR.